src/project.cc

```diff
@@ -123,6 +134,8 @@ struct ProjectProcessor {
             ok |= lang == LanguageId::ObjC;
           else if (A.consume_front("%objective-cpp "))
             ok |= lang == LanguageId::ObjCpp;
+          else if (A.consume_front("%cu "))
```

Comment:
`%cu ` is not necessary. CUDA is C++ (`clang::driver::types::isCXX` used in `lookupExtension`)